lib/go-atscfg/hostingdotconfig.go

```diff
@@ -196,6 +199,9 @@ func MakeHostingDotConfig(
 					if !topoHasServer {
 						continue
 					}
+					if !strings.Contains(string(*ds.Type), "LIVE") {
```

Comment:
Nitpick: better to use `https://github.com/apache/trafficcontrol/blob/master/lib/go-tc/enum.go#L870`

`if !ds.Type.IsLive()`